src/stactools/naip/stac.py

```diff
@@ -121,30 +123,45 @@ def create_item(
         )
 
     if fgdc_metadata_href is not None:
-        fgdc_metadata_text = read_text(fgdc_metadata_href)
-        fgdc = parse_fgdc_metadata(fgdc_metadata_text)
-    else:
-        fgdc = {}
+        if int(year) == 2020:
+            file_xpath = """gmd:identificationInfo/gmd:MD_DataIdentification/gmd:citation/
+            gmd:CI_Citation/gmd:title/gco:CharacterString"""
+
+            with fsspec.open(fgdc_metadata_href) as file:
+                root = XmlElement(
+                    etree.parse(file, base_url=fgdc_metadata_href).getroot()
+                )
+                resource_desc = root.find_text_or_throw(
+                    "".join(file_xpath.split()), missing_element("File Identifier")
+                )
+                if resource_desc is not None:
+                    dt = str_to_datetime(resource_desc.split(".")[0].split("_")[-1])
+                else:
+                    resource_desc, dt = get_id_date(cog_href)
+
+        elif int(year) < 2020:
```

Comment:
same, no need to convert to an int, you can just lexigraphically compare the strings.